src-types/src/entity.rs

```diff
@@ -31,39 +32,68 @@ impl Entity {
     }
 
     /// Gets a reference to a component or group of components from the Entity. These are for reading only.
-    fn get<'s, T : ComponentGroup + 'static>(&'s self) -> Option<<T as ComponentGroup>::Ref<'s>>  {
+    /// /// Example:
+    /// ```
+    /// if let Some((a, b)) = entity.get::<(A, B)>() {
```

Comment:
Unfortunately, these tests will not pass if we don't have the correct imports and setup. So when I run `cargo test` these fail. I remember we talked about removing these. Do you still want them here? Maybe there is a better place for these sorts of tests under example or below in the testing module? 

I'll tag the other ones with TAG: TEST